#Message that a robot receive after a replanning
int16 sender
int16[] destinations
Intlist[] new_Hole_paths
History[] listOfH
